containers/base/Dockerfile

```diff
@@ -15,98 +15,97 @@
 FROM debian:jessie
 MAINTAINER Google Cloud DataLab
 
+# Directory "py" may be empty and in that case it will git clone pydatalab from repo
+ADD pydatalab /datalab/lib/pydatalab
```

Comment:
These two lines should be after all of the dependency installation, since these are the bits most likely to change (and thus, require rebuilding every layer that comes after them).